lib/router.js

```diff
@@ -36,10 +36,15 @@ Cherrytree.prototype.initialize = function (options) {
   this.middleware = []
   this.options = _.extend({
     interceptLinks: true,
-    logError: true
+    logError: true,
+    Promise: window.Promise
```

Comment:
Specifically the node env support is just during webpack right? Then `global.Promise` should be fine. Also the actual webpack command should run in CI to catch this :)